python/test/test_tests.py

```diff
@@ -18,7 +18,6 @@ def test_TODOs():
     assert str(dp.smoothed_max_divergence()) == 'SmoothedMaxDivergence'
 
     # TODO: Add meaningful tests of partial constructors:
-    dp.m.then_private_expr(None, None) # type: ignore[arg-type]
```

Comment:
Maybe there's a change to the generated code and this method is no longer available? Otherwise, this test seems like it could stay around.